.github/workflows/ci.yaml

```diff
@@ -52,7 +52,16 @@ jobs:
             -   name: "Checkout"
                 uses: actions/checkout@v2
 
+            -   name: "Create cache dir"
+                run: mkdir .cache
+
+            -   name: "Cache DOCtor-RST"
+                uses: actions/cache@v1
+                with:
+                    path: .cache
+                    key: ${{ runner.os }}-doctor-rst-34
```

Comment:
I would prefer using the base branch programmatically. IIRC GithubActions exposes an env var

Reply:
I used `${{ github.base_ref || github.ref }}`.  `github.base_ref` is only available on pull requests. With `github.ref` I'm not 100% sure if it's contains only the branch name.